packages/react/src/publication/adapters/ReactionPresenter.ts

```diff
@@ -50,6 +55,33 @@ export class ReactionPresenter implements IReactionPresenter {
     });
   }
 
+  async revert(request: ReactionRequest): Promise<void> {
+    const statKey = getReactionStatKey(request.reactionType);
+
+    this.publicationCacheManager.update(request.publicationId, (current) => {
+      invariant(
+        isContentPublication(current),
+        `Reactions are not supported on ${current.__typename}`,
+      );
+
+      const removedStatKey = this.previousReactionType
```

Comment:
would that be possible for `this.previousReactionType` to be not defined? 
Wondering if there is some way we could guard ourselves to catch broken wiring faster.
As right now if you revert back the changes in `useReactionControllrer` (aka remove `useMemo`) you won't get any error and the code would just work but at the same time be broken.